integration-tests/test-app/account.ts

```diff
@@ -0,0 +1,18 @@
+import { registerHtml, useUrlParams, TramOneComponent } from '../../src/tram-one';
+
+const html = registerHtml();
+
+/**
+ * component to test url parameters
+ */
+const account: TramOneComponent = () => {
```

Comment:
This is a pattern components can use, to verify that they follow the interface for a "TramOneComponent"